cfme/automate/explorer/instance.py

```diff
@@ -196,10 +196,15 @@ def domain(self):
 
     @property
     def tree_path(self):
+        if self.appliance.version < '5.8':
```

Comment:
No need for this, as I understood we don't support 5.7 now

Reply:
Yeah, I will remove it